client-sdk/go/config/default.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+// DefaultNetworks is the default config containing known networks.
+var DefaultNetworks = Networks{
+	Default: "mainnet",
+	All: map[string]*Network{
+		// Mainnet network parameters.
+		// See https://docs.oasis.dev/general/oasis-network/network-parameters.
+		"mainnet": {
+			ChainContext: "53852332637bacb61b91b6411ab4095168ba02a50be4c3f82448438826f23898",
+			RPC:          "grpc.oasis.dev:443",
```

Comment:
resposting as this was lost in a file move:

        does the grpc-web proxy lets us dial it from the normal grpc client? or is there an additional thing on that server that handles normal grpc?